src/main/resources/portal.properties.EXAMPLE

```diff
@@ -6,6 +6,8 @@ db.user=cbio_user
 db.password=somepassword
 db.driver=com.mysql.jdbc.Driver
 db.connection_string=jdbc:mysql://localhost:3306/cbioportal?useSSL=false
+# Connection string for when both python and java connections are used in the docker entrypoint. When a connection URL is given for python, it will be used to form a connection to the mySQL database instead of the standard connection string. This allows for different connection properties to be used (like ssl connection for example). 
+# db.connection_string_python= jdbc:mysql://cbioDB:3306/cbioportal
```

Comment:
The comment on precedence should also  be made in the portal.properties-reference.md